internal/k8s/client.go

```diff
@@ -62,7 +62,10 @@ func (k KubectlClient) OpenService(ctx context.Context, lb LoadBalancer) error {
 
 	if k.env == EnvMinikube {
 		logger.Get(ctx).Infof("Waiting on minikube to load service: %s", lb.Name)
-		cmd := exec.CommandContext(ctx, "minikube", "service", lb.Name, "--url")
+
+		intervalSec := "1" // 1s is the smallest polling interval we can set :raised_eyebrow:
```

Comment:
i'm sad that this isn't the actual emoji but that's probably for the best

Reply:
the official name is :face_with_one_eyebrow_raised:, but github doesn't support unicode 10 yet :sob: